### DISCUSSION TITLE
Mini event : Mimic ou gros objet ?

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#2484] [Nouvelles idées]
Lors de notre voyage au bord de la route se trouve un coffre. Il est ornementé de belles décorations et ne semble appartenir à personne.

Deux possibilités : 

- L'ouvrir
- Le laisser et reprendre sa route

Si on l'ouvre on a au choix deux possibilités : 
- On obtient un objet légendaire ou +
- On prend une mort instantanée (99% de proba)

**Comment by Voisin1234** [comment]
Super idée pour les petits nouveaux qui n'ont pas beaucoup de points à perdre si il tombe sur la mort🤭

**Comment by romain22222** [comment]
Quand je parle de mimic bien sur, c'est au choix :
- un coffre piégé (engin explosif, bombe toxique reçue en plein visage, et j'en passe les testeurs et rédacteurs trouveront bien des inspirations)
- Piège autour du coffre
- Un véritable mimic (qui sait on a bien une sorcière, pourquoi pas un mimic)

**Reply by BastLast** [reply]
Omg romain a repris la drogue

**Reply by romain22222** [reply]
NYAH J Y TOUCHE PLUS DEPUIS LA DERNIERE PHASE DE CODE !
